tests/base.py

```diff
@@ -38,6 +38,8 @@
 ROCEV2_UNSUPPORTED_DEVS = {MLNX_VENDOR_ID: [CX3Pro_MLNX_PART_ID,
                                             CX3_MLNX_PART_ID]}
 
```

Comment:
please surround top-level functions and class definitions with two blank lines (PEP conventions)

Reply:
OK, I will fix it.